src/readme.ts

```diff
@@ -58,6 +58,13 @@ The Maven package is available at [https://mvnrepository.com/artifact/${packageI
 </dependency>
 \`\`\`
 
+
+### Go
+
+The go package is generated into the [\`github.com/hashicorp/cdktf-provider-${providerName}-go\`]https://github.com/hashicorp/cdktf-provider-${providerName}-go) package.
+
+\`go get github.com/hashicorp/cdktf-provider-${providerName}-go\`
```

Comment:
```suggestion
\`go get github.com/hashicorp/cdktf-provider-${providerName}-go/${providerName}\`
```

see here: https://github.com/hashicorp/terraform-cdk/issues/723#issuecomment-1189473189